authentication/auth.js

```diff
@@ -16,7 +16,7 @@ export default function Authentication(props) {
   }
   useEffect(() => {
     async function checkAuthentication() {
-      setAuthenticatedRef(getAuthToken(false))
+      setAuthenticatedRef(getAuthToken())
```

Comment:
I know the getAuthTokens in the rest of the app should have their falses removed, but I'm not sure if I was supposed to remove it here too.